account/src/main/java/com/caldevsupplychain/account/service/AccountServiceImpl.java

```diff
@@ -1,28 +1,27 @@
 package com.caldevsupplychain.account.service;
 
-import java.util.List;
```

Comment:
looks like your intellij import order is not updated

Reply:
fixed